core/field.ts

```diff
@@ -373,7 +373,7 @@ export abstract class Field<T = any>
     this.textElement_ = dom.createSvgElement(
       Svg.TEXT,
       {
-        'class': 'blocklyText',
+        'class': 'blocklyFieldText',
```

Comment:
```suggestion
        'class': 'blocklyText blocklyFieldText',
```

We want this class to be additional, not replacing! (Sorry I know this is different than some of the other issues)